tests/test_local.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
- Can we also add another test which validates the order of sample_id ? Or you can change the existing test case and capture the sample_id in the list and then assert it ?
- Another test is to iterate through Torch Dataloader to ensure it works.

Reply:
done x2